composites/Plugin/Shared/components/HelpText.js

```diff
@@ -1,30 +1,42 @@
 /* External dependencies */
-import React from "react";
+import React, { PureComponent } from "react";
 import PropTypes from "prop-types";
 import styled from "styled-components";
 
+/* Internal dependencies */
+import colors from "../../../../style-guide/colors";
+
 const YoastHelpText = styled.p`
-	font-size: 13px;
+	color: ${ props => props.color };
+	font-size: ${ props => props.fontSize };
```

Comment:
React renders this `fontSize` prop in the HTML as an attribute `font-size="0.8em"`. Renaming the prop to something else e.g. `textFontSize` would fix it.